ci/build_docs.sh

```diff
@@ -21,9 +21,12 @@ conda activate docs
 
 rapids-print-env
 
-export RAPIDS_VERSION="$(rapids-version)"
-export RAPIDS_VERSION_MAJOR_MINOR="$(rapids-version-major-minor)"
-export RAPIDS_DOCS_DIR="$(mktemp -d)"
+RAPIDS_VERSION="$(rapids-version)"
+RAPIDS_VERSION_MAJOR_MINOR="$(rapids-version-major-minor)"
+RAPIDS_DOCS_DIR="$(mktemp -d)"
+export RAPIDS_VERSION
+export RAPIDS_VERSION_MAJOR_MINOR
+export RAPIDS_DOCS_DIR
```

Comment:
```suggestion
RAPIDS_VERSION="$(rapids-version)"
export RAPIDS_VERSION

RAPIDS_VERSION_MAJOR_MINOR="$(rapids-version-major-minor)"
export RAPIDS_VERSION_MAJOR_MINOR

RAPIDS_DOCS_DIR="$(mktemp -d)"
export RAPIDS_DOCS_DIR
```

Very minor style thing, do what you want with it ... but if we're going to separate assigning and exporting like this, I think I'd prefer to do them one variable at a time, instead of having so much distance between the assignment and export calls for each variable. I find that form a bit easier to read.

Reply:
yeah, I had this originally and then changed it, but I agree that having the assignment and export calls closer together makes sense.